src/store/hooks.ts

```diff
@@ -39,3 +42,9 @@ export const useServerStore = () => {
   const oscap = useOscapData();
   return { ...oscap };
 };
+
+export const useBetaFlag = (flag: string): boolean => {
+  const firstbootFlag = useFlag(flag);
```

Comment:
If I understand correctly, this hook will be re-usable outside of first boot, right? Could you give the const more universal name? Like `unleashFlag` or something like that?